### ISSUE TITLE
Adhering to colour contrast accessibility guidelines

### ISSUE THREAD

**Issue by gregdorward** [redbadger/website-honestly#431]
**See last comments for the resolution**

**Description of the issue:**
There are currently a number of places throughout Website Honestly where we are failing to meet accessibility guidelines for sufficient colour contrasting. The guidelines state that when using text under 19 pixels in size, a contrast of 4.5:1 should exist.

With Badger Red (#FC1D42), these are our current contrasts:

#FC1D42 against a background of #F8F8F8 - contrast of 3.67:1
#FC1D42 against a background of #212121 - contrast of 4.14:1

To fix this, we will either need to come up with 2 new types of red to use (one darker, one lighter), or increase the font size wherever this colour is used to 19 pixels

**Browser used:**
Chrome

**OS used:**
OS Sierra

**Device used:**
Macbook

**Steps to reproduce:**

1. Navigate to Website Honestly
2. Run an accessibility check on each page (e.g. WAVE)

**Actual result:**
As described above, we're currently falling short of some accessibility standards

**Expected result:**
Colour contrast of at least 4.5:1 should exist throughout the site

**Comment by gregdorward** [comment]
cc @sarigriffiths & @Roisi

**Comment by gregdorward** [comment]
Below are two potential options for darkening and lightening the reds respectively. These reds look to be just within the range of what's acceptable, but may not be right at the limit of what we can get away with

#e5051a against a background of #f8f8f8

<img width="966" alt="screen shot 2017-04-21 at 09 53 05" src="https://cloud.githubusercontent.com/assets/20296527/25270361/56772bd4-2678-11e7-81d9-3d5f743c750e.png">

#ff316b against a background of #212121

<img width="1008" alt="screen shot 2017-04-21 at 09 51 56" src="https://cloud.githubusercontent.com/assets/20296527/25270336/2e14e500-2678-11e7-9fab-d0f8ced056cf.png">

**Comment by sarigriffiths** [comment]
Happy with the darker red (#e5051a) for the light grey background.
Can we change the lighter red for the dark background to be: #fc3a54? Making it less purple. And let's keep #212121 grey.

Just wondering - why Sentinel font not rendering on those screenshots? They are Georgia ^. @gregdorward

**Comment by gregdorward** [comment]
Not sure why it's not rendering the Sentinel font @sarigriffiths . It could be because these screenshots were taken from within a WAVE accessibility report, so it could be that they render it differently. I'll see if I can find out

**Comment by gregdorward** [comment]
@sarigriffiths the font I see when I load this page is 'ProximaNova-Light,sans-serif', is that not right? I think Typography has been flaky again recently (it was down a few minutes ago), so that must have been our fallback font. This is what it looks like now:

<img width="481" alt="screen shot 2017-04-24 at 10 46 12" src="https://cloud.githubusercontent.com/assets/20296527/25331531/547fa03a-28db-11e7-95b0-821d6992d9a2.png">

**Comment by gregdorward** [comment]
Screenshots of #e5051a (against a dark background) and #fc3a54 (against a light background)

<img width="513" alt="screen shot 2017-04-24 at 10 48 36" src="https://cloud.githubusercontent.com/assets/20296527/25331630/a602b38e-28db-11e7-9b4a-17af923d4caa.png">

<img width="507" alt="screen shot 2017-04-24 at 10 49 15" src="https://cloud.githubusercontent.com/assets/20296527/25331631/a61d1058-28db-11e7-9734-6762aee80e6e.png">

**Comment by sarigriffiths** [comment]
👍 Thanks. Go ahead please.

**Comment by gregdorward** [comment]
This one is now fixed, with no shortfalls in contrasting on website honestly and the blog
